content/source/docs/cloud/sentinel/manage-policies.html.md

```diff
@@ -64,18 +64,72 @@ Terraform Cloud has support for Sentinel's modules feature. This allows you to w
 
 ```hcl
-module "foo" {
-    source = "./modules/foo.sentinel"
+module "timezone" {
+    source = "./modules/timezone.sentinel"
 }
 ```
 
-The following entry would tell a policy check to load the code at `./modules/foo.sentinel` relative to the policy set working directory and make it available to be imported with a statement such as `import "foo"` at the top of your Sentinel policy code. This module will be available to all of the policies within the policy set.
+Create a `modules` directory within the policy set working directory, and create a `timezone.sentinel` that looks as follows:
+
+```sentinel
+import "http"
+import "json"
+import "strings"
+
+uri = "https://raw.githubusercontent.com/dmfilipenko/timezones.json/master/timezones.json"
+request = http.get(uri)
+response = json.unmarshal(request.body)
+
+offset = func(abbr) {
+	timezone = filter response as _, r {
+		r.abbr is strings.to_upper(abbr)
+	}
+	for timezone as tz {
+		print("Getting timezone data for", tz.value)
+		print("Details:", tz.text)
+		print("Abbreviation:", tz.abbr)
+		print("Offset:", tz.offset)
+		return tz.offset
+	}
+}
+```
 
--> **NOTE:** At this point in time, you cannot load modules from outside of policy set working directory hierarchy. This means in the above example, a `source` of `../modules/foo.sentinel` will not work.
+The above configuration would tell a policy check to load the code at `./modules/timezone.sentinel` relative to the policy set working directory and make it available to be imported with the statement `import "timezone"` at the top of your Sentinel policy code. This module will be available to all of the policies within the policy set.
+
+-> **NOTE:** At this point in time, you cannot load modules from outside of policy set working directory hierarchy. This means in the above example, a `source` of `../modules/timezone.sentinel` will not work.
 
 ### Sentinel policy code files
 
-Sentinel policies themselves are defined in individual files (one per policy) in the same directory as the `sentinel.hcl` file. These files must match the name of the policy from the configuration file and carry the `.sentinel` suffix. Using the configuration example above, a policy file named `sunny-day.sentinel` should also exist alongside the `sentinel.hcl` file to complete the policy set.
+Sentinel policies themselves are defined in individual files (one per policy) in the same directory as the `sentinel.hcl` file. These files must match the name of the policy from the configuration file and carry the `.sentinel` suffix. Using the configuration example above, a policy file named `enforce-terraform-maintenance-windows.sentinel` should also exist alongside the `sentinel.hcl` file to complete the policy set. 
+
+Using the `enforce-terraform-maintenance-windows.sentinel` policy as an example, we can use the `time` and `tfrun` imports along with our custom `timezone` module to enforce checks that:
+
+1. Load the time when the Terraform Run occurred
```

Comment:
```suggestion
1. Load the time when the Terraform run occurred
```

Normal nouns like "run" shouldn't be capitalized unless they function as brand names.